app-frontend/src/app/pages/projects/edit/annotate/annotate.controller.js

```diff
@@ -374,7 +396,7 @@ export default class AnnotateController {
         this.disableSidebarAction = true;
         this.disableToolbarAction = true;
         this.getMap().then((mapWrapper) => {
-            this.createEditableAndDraggableDrawLayer(mapWrapper, geometry, false);
+            this.createEditableAndDraggableDrawLayer(mapWrapper, geometry);
```

Comment:
this function name probably would be fine as `createEditableDrawLayer`, since we aren't creating editable but non-draggable layers anywhere